### PR TITLE
Adjusting code snippets

### PR DESCRIPTION
**Pull request by AderbalFarias**
Adjusting the indentation of the code snippets

### CODE REVIEW

**Comment by illfated** [comment]
Looks quite OK to me. Just out of curiosity, is the `<appSettings>` section XML code, or something else?
If it actually is XML code, could we add a MarkDown backtick code fence around it along with syntax highlighting for XML?

**Comment by eross-msft** [comment]
@billmath Please confirm if this text is okay to merge. Thanks!!

**Comment by illfated** [comment]
@AderbalFarias : Would you happen to know if the `<appSettings>` section is XML or if it is defined as something else?

**Comment by AderbalFarias** [comment]
@illfated it is XML

File: WindowsServerDocs/identity/ad-fs/development/Single-Page-Application-with-AD-FS.md [hunk 1]
```diff
@@ -150,10 +151,10 @@ and add:
 Change the appsettings as below:
```
Comment (illfated) [inline WindowsServerDocs/identity/ad-fs/development/Single-Page-Application-with-AD-FS.md]:
```suggestion
```xml
```

File: WindowsServerDocs/identity/ad-fs/development/Single-Page-Application-with-AD-FS.md [hunk 2]
```diff
@@ -150,10 +151,10 @@ and add:
...
 
     <appSettings>
-    <add key="ida:Audience" value="https://localhost:44326/" />
-    <add key="ida:AdfsMetadataEndpoint" value="https://fs.contoso.com/federationmetadata/2007-06/federationmetadata.xml" />
-    <add key="ida:Issuer" value="https://fs.contoso.com/adfs" />
-      </appSettings>
+        <add key="ida:Audience" value="https://localhost:44326/" />
+        <add key="ida:AdfsMetadataEndpoint" value="https://fs.contoso.com/federationmetadata/2007-06/federationmetadata.xml" />
+        <add key="ida:Issuer" value="https://fs.contoso.com/adfs" />
+    </appSettings>
```
Comment (illfated) [inline WindowsServerDocs/identity/ad-fs/development/Single-Page-Application-with-AD-FS.md]:
```suggestion
    </appSettings>
    ```
```

**Comment by illfated** [comment]
Feel free to choose for yourself if the block indentation for the XML section can be reduced / removed.

**Comment by billmath** [comment]
#sign-off

**Comment by illfated** [comment]
@AderbalFarias : Would you be able to commit my suggested additions?

**Pull request merged**